WalletWasabi.Tests/UnitTests/Hwi/MockedDeviceTests.cs

```diff
@@ -562,7 +563,7 @@ public async Task JadeMockTestsAsync(Network network)
 		IEnumerable<HwiEnumerateEntry> enumerate = await client.EnumerateAsync(cts.Token);
 		HwiEnumerateEntry entry = Assert.Single(enumerate);
 		Assert.Equal(HardwareWalletModels.Jade, entry.Model);
-		Assert.True(HwiValidationHelper.ValidatePathString(entry.Model, "COM3"));
+		Assert.True(HwiValidationHelper.ValidatePathString(entry.Model, "COM3", OSPlatform.Windows));
```

Comment:
Remove this entirely or add a method called `GetPlatformSpecificDevicePath` which will return an example of what would a device give back on a specific platform.